src/workflow_components/workflow.py

```diff
@@ -200,7 +252,9 @@ def from_dict(obj_dict: Dict[str, Any]) -> "Workflow":
                         f"[-] Couldn't find the triggering workflow '{workflow_name}' in repository '{repo}'"
                     )
                 else:
-                    w_triggering = get_or_create_workflow(w_path)
+                    w_triggering = get_or_create_workflow_run_workflow(
+                        *UsesString.split_path_and_ref(w_path)
```

Comment:
Is it should be a pointer in this case?

Reply:
It is not a pointer, it receives a tuple and unpacks it inside the `get_or_create_workflow_run_workflow` func call in order to pass two parameters to it instead of one list.